### DISCUSSION TITLE
What is the tallest mountain 🗻  in the world ?

### DISCUSSION THREAD

**Discussion by Sriansh-raj** [Sriansh-raj/GitHub-Achievements#21] [Galaxy Qs] [answered]
What is the tallest mountain in the world ?

**Comment by bytesleo** [comment, marked as answer]
The tallest mountain in the world is Mount Everest. It is located in the Himalayas on the border of Nepal and China (Tibet Autonomous Region). Mount Everest stands at an elevation of 8,848.86 meters (29,031.7 feet) above sea level. It is a popular destination for mountaineers and has been a subject of fascination and exploration for centuries.

**Reply by Sriansh-raj** [reply]
Congratulations 🥳 @kevoj you just earned a prestigious `Galaxy Brain` achievement.
Now pls give some time and enter your name in [Users_who_got_Galaxy_Brain.md](https://github.com/Sriansh-raj/GitHub-Achievements/blob/main/Users_who_got_Galaxy_Brain.md) to make a history 🎉
